Lib/test/test_dict.py

```diff
@@ -1176,6 +1176,48 @@ def iter_and_mutate():
 
         self.assertRaises(RuntimeError, iter_and_mutate)
 
```

Comment:
do we need to test order for dict comprehensions ?

Reply:
Yes, thanks!